TreasureDataExampleSwift/Podfile.lock

```diff
@@ -0,0 +1,25 @@
+PODS:
```

Comment:
Maybe we should consider removing/merging TreasureDataExampleSwift on some cleanup PR later. Just make it entirely Swift, which I believe covers a majority of audiences (just a self-note, we probably don't need to make it in this PR).

Reply:
I agree that 2 demo apps is redundant. I suggest retaining two code examples (objective-c and swift) and merged the UI into a single example app.